File: 2425/C++/secondaEsercitazione.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    string magic_word = "hello";
    string try_word = ""; 
    int numero_tentativi = 0;

    while(numero_tentativi < 3){
        cout << "Inserisci la parola magica" << endl;
        cin >> try_word;
        if(magic_word == try_word){
            cout << "Hai vinto in " << (numero_tentativi + 1) << " tentativi" << endl;
            break;
        }
        else{
            cout << "Ritenta sarai più fortunato!!!" << endl;
            numero_tentativi++;
        }

    }

    if(numero_tentativi < 3){
        cout << "Hai superato il prmo livello" << endl;
        cout << "Secondo livello" << endl;
    }
    else{
        cout << "Hai perso!!!" << endl;
    }

    return 0;
}

File: 2425/Python/secondaEsercitazione.py
'''
Indovina la parola magica
Hai tre tentativi per indovinare una
parola magica 
'''
magic_word = "hello"

numero_tentativi = 0

while(numero_tentativi < 3):
    print("Inserisci la parola magica")
    try_word = input()
    if (magic_word == try_word):
        print("Hai vinto in", int(numero_tentativi + 1) ,"tentativi")
        break
    else:
        print("ritenta")
        numero_tentativi = numero_tentativi + 1

print("Primo quadro superato")
print("Hai ancora ", int(3 - numero_tentativi), " vite")


'''
for i in range(3):
    print("Inserisci la parola magica")
    try_word = input()
    if (magic_word == try_word):
        print("Hai vinto in", i+1 ," tentativi")
        break
    else:
        print("ritenta")
'''